pandas/core/generic.py

```diff
@@ -3929,36 +3929,38 @@ def pipe(self, func, *args, **kwargs):
         return com._pipe(self, func, *args, **kwargs)
 
     _shared_docs['aggregate'] = ("""
-    Aggregate using callable, string, dict, or list of string/callables
+    Aggregate using one or multiple operations along the specified axis.
```

Comment:
multiple -> more

Reply:
along -> over I think.

Does along mean apply to each element in an axis? And over means collapse an axis?